### PR TITLE
Improve Quebec Payroll

### PR DESCRIPTION
**Pull request by ghost**
This module adds the Quebec payroll structure. The following pull request needs to be merged before:
#43

### CODE REVIEW

File: l10n_ca_qc_hr_payroll/__openerp__.py [hunk 1]
```diff
@@ -31,6 +30,8 @@
 
 Contributors
 ------------
+* David Dufresne <david.dufresne@savoirfairelinux.com>
+* Pierre Lamarche <pierre.lamarche@savoirfairelinux.com>
```
Comment (bwrsandman) [inline l10n_ca_qc_hr_payroll/__openerp__.py]: Not a big deal, but the order of contributor is ascending, with the newer ones at the bottom, not the top

File: l10n_ca_qc_hr_payroll/README.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Install the following requirements:
+
+ - libdmtx-utils: a library for generating DataMatrix barcodes
+    - On ubuntu: sudo apt-get install dmtx-utils
+    - For other operating systems, refer to: http://www.libdmtx.org/
+
+ - pydmtx: the python wrapper around libdmtx
+    - git clone git://libdmtx.git.sourceforge.net/gitroot/libdmtx/dmtx-wrappers
+    - cd dmtx-wrappers/python/
+    - sudo python setup.py install
```
Comment (max3903) [inline l10n_ca_qc_hr_payroll/README.rst]: Please replace pydmtx (obsolete) with elaphe

File: .travis.yml [hunk 3]
```diff
@@ -29,6 +29,10 @@ install:
   - git clone --depth=1 https://github.com/OCA/account-payment.git ${HOME}/account-payment -b ${VERSION}
   - git clone --depth=1 https://github.com/OCA/bank-statement-reconcile.git ${HOME}/bank-statement-reconcile -b ${VERSION}
   - git clone --depth=1 https://github.com/OCA/webkit-tools.git ${HOME}/webkit-tools -b ${VERSION}
+  - sudo apt-get install dmtx-utils
+  - git clone git://libdmtx.git.sourceforge.net/gitroot/libdmtx/dmtx-wrappers ${HOME}/dmtx-wrappers
+  - cd ${HOME}/dmtx-wrappers/python/
+  - sudo python setup.py install
```
Comment (max3903) [inline .travis.yml:47]: Please remove this as you use elaphe

File: l10n_ca_qc_hr_payroll/hr_releve_1_other_info.py [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+        ),
+        'amount': fields.float(
+            'Amount',
+            digits=(15, 2),
+            required=True,
+        ),
+    }
+
+
+class hr_releve_1_other_info_source(orm.Model):
```
Comment (bwrsandman) [inline l10n_ca_qc_hr_payroll/hr_releve_1_other_info.py]: Can you split this into two files, one for each model

Comment (ghost) [inline l10n_ca_qc_hr_payroll/hr_releve_1_other_info.py]: @bwrsandman Sure, Thanks!

File: .travis.yml (hunk 3 above)
Comment (ghost) [inline .travis.yml:47]: Done!

**Comment by max3903** [comment]
Please tell Travis to install elaphe and Travis should be green.
Runbot is missing https://github.com/OCA/bank-statement-reconcile.git

**Comment by bwrsandman** [comment]
Needs a rebase

**Comment by bwrsandman** [comment]
Rebase this branch ~~after merge of #68~~

**Comment by ghost** [comment]
@max3903 You may remove needs fixing label here as well

File: l10n_ca_qc_hr_payroll/models/hr_releve_1_summary.py [hunk 5]
```diff
@@ -0,0 +1,582 @@
...
+        'qpip_amount_ee': 0.0,
+        'qpip_amount_er': 0.0,
+        'qit_amount_1': 0.0,
+        'sub_total_remitted': 0.0,
+        'hsf_total_wage_bill': 0.0,
+        'hsf_salaries': 0.0,
+        'hsf_contribution_rate': 0.0,
+        'hsf_amount_remitted': 0.0,
+        'cnt_salaries': 0.0,
+        'cnt_rate': 0.0,
```
Comment (bwrsandman) [inline l10n_ca_qc_hr_payroll/models/hr_releve_1_summary.py]: In here twice

File: l10n_ca_qc_hr_payroll/models/hr_releve_1_summary.py [hunk 6]
```diff
@@ -0,0 +1,582 @@
...
+        'wsdrf_salaries': 0.0,
+        'wsdrf_rate': 0.0,
```
Comment (bwrsandman) [inline l10n_ca_qc_hr_payroll/models/hr_releve_1_summary.py]: In here twice

File: l10n_ca_qc_hr_payroll/report/__init__.py [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+import report_releve_1
```
Comment (bwrsandman) [inline l10n_ca_qc_hr_payroll/report/__init__.py]: relative import needed

File: l10n_ca_qc_hr_payroll/README.rst [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+It allows you to:
+ - create the payslips of your employees
+ - produce the Relevé 1 for Revenu Québec
+
+Installation
+============
+
+You need to install the elaphe python library (a barcode generator):
+
+	- sudo pip install elaphe
```
Comment (bwrsandman) [inline l10n_ca_qc_hr_payroll/README.rst]: Need to add `pip install elaphe` to .travis

**Comment by bwrsandman** [comment]
Depends on:
- [ ] Improve Canada Payroll #43 
  - [x] Partner aging OCA/account-invoice-reporting#32

**Pull request closed without merging**